Makefile

```diff
@@ -14,14 +14,34 @@ clean: ## delete generated byte code and coverage reports
 	rm -rf assets
 	rm -rf pii_report
 
+piptools: ## install pinned version of pip-compile and pip-sync
+	pip install -r requirements/pip_tools.txt
+
+## The following defines a string constant that contains a multi-line comment.
+define COMMON_CONSTRAINTS_TEMP_COMMENT
+# This is a temporary solution to override the real common_constraints.txt\n# In edx-lint, until the pyjwt constraint in edx-lint has been removed.\n# See BOM-2721 for more details.\n# Below is the copied and edited version of common_constraints\n
+endef
```

Comment:
Is this still necessary, considering https://github.com/edx/edx-lint/commit/4ee5f3a5e5cffa9294d5d499cd610cdd4cf7fd4c @Jawayria

Reply:
@jinder1s: I _think_ the point is to remove the Django constraint? But, the rest of the file doesn't need to be so out of date.